### DISCUSSION TITLE
Lambda Annotations & CDK (is there an update?)

### DISCUSSION THREAD

**Discussion by mncoldsmoke** [aws/aws-lambda-dotnet#1507] [General]
In https://github.com/aws/aws-lambda-dotnet/discussions/1230 it was implied that smooth integration with Lambda functions and the CDK was on the roadmap. Has there been any progress?

**Comment by ashishdhingra** [comment]
@mncoldsmoke As of now, the team is working on other high priority feature work and there are no concrete timelines by which CDK would be implemented.

**Reply by dguisinger** [reply]
Its now been a year since this discussion was closed, and two years since the original discussion indicated it was on the roadmap.

Any progress on this getting prioritized?  AWS is really pushing a CDK-centric world.
